Libs/MRML/Widgets/qMRMLDisplayNodeWidget.cxx

```diff
@@ -252,6 +256,31 @@ void qMRMLDisplayNodeWidget::setSliceIntersectionThicknessVisible(bool visible)
 }
 
 //------------------------------------------------------------------------------
+void qMRMLDisplayNodeWidget::setSliceIntersectionOpacity(double opacity)
+{
+  Q_D(qMRMLDisplayNodeWidget);
+  if (!d->MRMLDisplayNode.GetPointer())
+  {
+    return;
+  }
+  d->MRMLDisplayNode->SetSliceIntersectionOpacity(opacity);
+}
+
+//------------------------------------------------------------------------------
+double qMRMLDisplayNodeWidget::sliceIntersectionOpacity()const
+{
+  Q_D(const qMRMLDisplayNodeWidget);
+  return d->SliceIntersectionOpacitySlider->value();
+}
+
+//------------------------------------------------------------------------------
+void qMRMLDisplayNodeWidget::setSliceIntersectionOpacityVisible(bool visible)
+{
+  Q_D(qMRMLDisplayNodeWidget);
+  d->SliceIntersectionOpacitySlider->setVisible(visible);
+}
+
+//------------------------------------------------------------------------------
 void qMRMLDisplayNodeWidget::updateWidgetFromMRML()
 {
   Q_D(qMRMLDisplayNodeWidget);
```

Comment:
Need to implement update of slice intersection opacity widget from MRML in updateWidgetFromMRML()